Windows/Debugger/Debugger_MemoryDlg.cpp

```diff
@@ -84,107 +96,130 @@ void CMemoryDlg::Update(void)
 {
 	if (m_hDlg != NULL)
 	{
-		CtrlMemView *mv = CtrlMemView::getFrom(GetDlgItem(m_hDlg,IDC_MEMVIEW));
-		if (mv != NULL)
-			mv->redraw();
+		if (memView != NULL)
+			memView->redraw();
 	}	
 }
 
+void CMemoryDlg::searchBoxRedraw(std::vector<u32> results) {
+
+	int index;
+	wchar_t temp[256];
+	//std::lock_guard<std::recursive_mutex> guard(lock_);
```

Comment:
What's the reason for leaving this commented out?

Reply:
I wasn't sure what this does, I took it from the symbol map when I was writing. Is it important?

Reply:
Since you are not accessing what this lock was protecting over in SymbolMap here, you can just delete this line.